README.md

```diff
@@ -118,6 +118,10 @@ Type: `string`. **Required.**
 
 Your module name. This is used to create the default [`searchPlaces`] and [`packageProp`].
 
+### cosmiconfigOptions
+
+Type: `Object`.
+
```

Comment:
I think you can delete this heading & section.

Reply:
done